tests/fuzz/wasm-mutator-fuzz/CMakeLists.txt

```diff
@@ -91,10 +91,7 @@ if (NOT DEFINED WAMR_BUILD_SIMD)
   set (WAMR_BUILD_SIMD 1)
 endif ()
 
-if (NOT DEFINED WAMR_BUILD_REF_TYPES)
-  # Disable reference types by default
-  set (WAMR_BUILD_REF_TYPES 0)
-endif ()
+set (WAMR_BUILD_REF_TYPES 1)
```

Comment:
Could we keep `if (NOT DEFINED WAMR_BUILD_REF_TYPES) set (WAMR_BUILD_REF_TYPES 1) endif ()`? Maybe we need to disable reference types some time in the future.